systemtests/tests/droplet-s3/testrunner

```diff
@@ -59,6 +59,14 @@ check_for_zombie_jobs storage=File
 check_two_logs
```

Comment:
Since you touched this file, you should also add the copyright notice.  Weird that pr-tool did not say anything here.   From what i can tell this file was introduced in 2023 (commit 839c71041167b67414b74fb74265b73387c202f6) so it should be 2023-2023 as well.

Reply:
yes make sense.